subprojects/configuration-cache/src/main/kotlin/org/gradle/configurationcache/fingerprint/ConfigurationCacheFingerprint.kt

```diff
@@ -78,7 +78,13 @@ sealed class ConfigurationCacheFingerprint {
         val value: Any?
     ) : ConfigurationCacheFingerprint()
 
-    data class RemoteScript(val uri: URI) : ConfigurationCacheFingerprint()
+    data class RemoteScript(
+        val uri: URI
+    ) : ConfigurationCacheFingerprint()
+
+    data class FileSystemDefaultExcludes(
```

Comment:
DefaultExcludes naming sounds too generic for me. So decided to have some meaningful prefix